.gitignore

```diff
@@ -140,3 +140,8 @@ dist
 .yarn/build-state.yml
 .yarn/install-state.gz
 .pnp.*
+
+
+# Docker
+database
+docker-compose.local.yml
```

Comment:
Any idea how to change so that contributors are not accidently pushing their credentials?